packages/react-components/react-components/src/unstable/index.ts

```diff
@@ -11,14 +11,21 @@ export type { AlertProps, AlertSlots, AlertState } from '@fluentui/react-alert';
 export {
   AvatarGroup,
   AvatarGroupItem,
+  AvatarGroupPopover,
+  AvatarGroupProvider,
   avatarGroupClassNames,
   avatarGroupItemClassNames,
+  avatarGroupPopoverClassNames,
   renderAvatarGroup_unstable,
   renderAvatarGroupItem_unstable,
+  renderAvatarGroupPopover_unstable,
   useAvatarGroup_unstable,
   useAvatarGroupItem_unstable,
+  useAvatarGroupPopover_unstable,
   useAvatarGroupStyles_unstable,
   useAvatarGroupItemStyles_unstable,
+  useAvatarGroupPopoverStyles_unstable,
+  useAvatarGroupContext_unstable,
```

Comment:
_nit:_ Order alphabetically

```suggestion
  AvatarGroup,
  AvatarGroupItem,
  AvatarGroupPopover,
  AvatarGroupProvider,
  avatarGroupClassNames,
  avatarGroupItemClassNames,
  avatarGroupPopoverClassNames,
  renderAvatarGroup_unstable,
  renderAvatarGroupItem_unstable,
  renderAvatarGroupPopover_unstable,
  useAvatarGroup_unstable,
  useAvatarGroupContext_unstable,
  useAvatarGroupItem_unstable,
  useAvatarGroupItemStyles_unstable,
  useAvatarGroupPopover_unstable,
  useAvatarGroupPopoverStyles_unstable,
  useAvatarGroupStyles_unstable,
```